honcho/export/base.py

```diff
@@ -19,17 +20,22 @@
 
 
 class BaseExport(object):
-    def __init__(self, template_env=None):
+    # Override this in subclasses -- e.g.: `supervisord`, `upstart`, etc.
+    default_template_dir = ''
+
+    def __init__(self, template_dir=None, template_env=None):
         if template_env is None:
             template_env = _default_template_env()
         self._template_env = template_env
+        self._template_dir = template_dir or self.default_template_dir
 
-    def get_template(self, path):
+    def get_template(self, name):
         """
         Retrieve the template at the specified path. Returns an instance of
         :py:class:`Jinja2.Template` by default, but may be overridden by
         subclasses.
         """
+        path = os.path.join(self._template_dir, name)
```

Comment:
Perhaps don't do this here, and just change the search path of the `FileSystemLoader` below?